src/child/index.js

```diff
@@ -20,6 +21,15 @@ require('script!../../node_modules/BitFlux/dist/bitflux.js');
 
 const store = configureStore();
 
+fin.desktop.main(() => {
+    fin.desktop.Window.getCurrent().addEventListener('bounds-changed', (e) => {
```

Comment:
My initial feeling is that this is a bit excessive - just a feeling though.

Rather than storing the dimensions whenever the window changes size, can this just be done when the user selects compact view (and store them as `previousExpandedDimensions`, or something similar)? Would this be sufficient (or is there another use case which I'm not considering)?